packages/aws-cdk-lib/aws-sns/test/sns.test.ts

```diff
@@ -235,13 +266,6 @@ describe('Topic', () => {
       PolicyDocument: {
         Version: '2012-10-17',
         Statement: [
-          {
-            'Sid': '0',
-            'Action': 'sns:*',
-            'Effect': 'Allow',
-            'Principal': { 'AWS': 'arn' },
-            'Resource': '*',
-          },
```

Comment:
Previously, the SSL policy was created after the `addToResourcePolicy` method was called. But now the SSL policy is created first, so this `Sid` by the `addToResourcePolicy` method will be 1, i.e. after the SSL policy.

This is NOT a breaking change, although it does cause a change in the CloudFormation template.

```
          {
            'Sid': '1',
            'Action': 'sns:*',
            'Effect': 'Allow',
            'Principal': { 'AWS': 'arn' },
            'Resource': '*',
          },
```